HexCtrl/HexCtrl.h

```diff
@@ -64,6 +64,7 @@ namespace HEXCTRL
 		[[nodiscard]] virtual auto GetPagePos()const->ULONGLONG = 0;         //Get current page a cursor stays at.
 		[[nodiscard]] virtual DWORD GetPageSize()const = 0;                  //Current page size.
 		[[nodiscard]] virtual auto GetSelection()const->std::vector<HEXSPAN> = 0; //Get current selection.
+		[[nodiscard]] virtual wchar_t GetUnprintableCharacter()const = 0;     //Set unprintable replacement character

```

Comment:
Verbosity. `GetUnprintableChar` would be absolutely fine.
Also the comment is misleading.